test/jdk/java/awt/print/PrinterJob/ValidatePage/ValidatePage.java

```diff
@@ -225,15 +222,14 @@ public ValidatePage() {
         panel.add(printButton);
         add(panel);
         TextArea ta = new TextArea(10, 45);
-        String ls = System.getProperty("line.Separator", "\n");
         ta.setText(
-                "When validating a page, the process is 1st to find the closest matching " + ls +
-                "paper size, next to make sure the requested imageable area fits within " + ls +
-                "the printer's imageable area for that paper size. Finally the top and " + ls +
-                "left margins will be shrunk if they are too great for the adjusted " + ls +
-                "imageable area to fit at that position. They will shrink by the minimum" + ls +
-                "needed to accomodate the imageable area." + ls + ls +
-                "To test 6229507, put the minimum margins (all 0s) in Page Setup dialog." + ls +
+                "When validating a page, the process is 1st to find the closest matching \n" +
+                "paper size, next to make sure the requested imageable area fits within \n" +
+                "the printer's imageable area for that paper size. Finally the top and \n" +
+                "left margins will be shrunk if they are too great for the adjusted \n" +
+                "imageable area to fit at that position. They will shrink by the minimum\n" +
+                "needed to accomodate the imageable area.\n \n \n" +
+                "To test 6229507, put the minimum margins (all 0s) in Page Setup dialog.\n" +
                 "Compare Imageable width, height, and margins of portrait against landscape.");
```

Comment:
```suggestion
                "When validating a page, the process is 1st to find the closest matching\n" +
                "paper size, next to make sure the requested imageable area fits within\n" +
                "the printer's imageable area for that paper size. Finally the top and\n" +
                "left margins will be shrunk if they are too great for the adjusted\n" +
                "imageable area to fit at that position. They will shrink by the minimum\n" +
                "needed to accomodate the imageable area.\n\n\n" +
                "To test 6229507, put the minimum margins (all 0s) in Page Setup dialog.\n" +
                "Compare Imageable width, height, and margins of portrait against landscape.");
```
The spaces at the end of lines are redundant.

Reply:
Spaces at the end of the line can be removed.